Commonalities/documentation/Working/CAMARA-AuthN-AuthZ-Concept.md

```diff
@@ -141,6 +148,97 @@ The authorization code is a temporary code that the client will exchange for an
 
 Proof Key for Code Exchange (PKCE is specified in RFC 7636) is a kind of proof of possession. It is an extension to the authorization code flow to prevent CSRF and authorization code injection attacks. The technique involves the client first creating a secret on each authorization request, and then using that secret again when exchanging the authorization code for an access token. This way if the code is intercepted, it will not be useful since the token request relies on the initial secret. PKCE uses cryptography to guarantee that the client exchanging an OAuth2 code for tokens is the same client that started the original OAuth2 request. It is hence important to use this extension when using Auth code grant flow.
 
+## Identification of UEs <a name="ueid"></a>
+
```

Comment:
A general introduction section with the possible UE identifiers (IP address etc.) would help set a scene. Then some lines on why some of these identifiers are considered more optimal than the others could help the reader get the context.

Reply:
Makes a lot of sense

Reply:
Feel free to close this after the introduction section is added.